polkadot/zombienet-sdk-tests/Cargo.toml

```diff
@@ -9,15 +9,15 @@ publish = false
 
 [dependencies]
 env_logger = "0.11.2"
-log = "0.4"
+log = { workspace = true }
 subxt = { version = "0.37", features = ["substrate-compat"] }
 subxt-signer = { version = "0.37" }
```

Comment:
All deps should be workspace deps. No dep should be crate local anymore.

Reply:
Sounds good, thanks for the feedback 👍